doc/tutorials/basic/layouts.md

```diff
@@ -0,0 +1,303 @@
+# Layout Content
+
+In this guide, we will learn how to layout, i.e., arrange, Python objects, including Panel components:
+
+- *Layouts* are available in the `pn` namespace.
+- Arrange Python objects, including Panel components, using [`pn.Column`](../../reference/layouts/Column.ipynb) and [`pn.Row`](../../reference/layouts/Row.ipynb).
```

Comment:
I thought it meant the same as layout. I used the term because its used in the *how-to* guides. I can now see that there we used it for *spacing*, *alignment* and *size*. I think we should stick to that use.

![image](https://github.com/holoviz/panel/assets/42288570/74d6a47b-c506-490c-89dd-0cddcdd8b624)

I will update.